src/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.tsx

```diff
@@ -1,7 +1,69 @@
+import { useTranslation } from 'react-i18next'
 import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+
+import useForm from '~/hooks/use-form'
+import { styles } from '~/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.styles'
+import SettingItem from '~/components/setting-item/SettingItem'
 
 const QuizSettingsContainer = () => {
-  return <Box>Quiz settings</Box>
+  const { t } = useTranslation()
+  const { data, handleInputChange } = useForm({
+    initialValues: {
+      pointValues: false,
+      scoredUnscoredResponses: false,
+      correctAnswers: false,
+      shuffleQuestions: false
+    }
+  })
+
+  return (
+    <Box>
+      <Box>
+        <Typography sx={{ ...styles.title, ...styles.topTitle }}>
```

Comment:
Use our spliceSx function to combine both styles:

`<Typography sx={spliceSx(styles.title, styles.topTitle)}`

Reply:
Thank you, I will use it